src/components/Composer/index.js

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+/* eslint-disable react/require-default-props */
```

Comment:
It requires defining defaultProps for each prop that isn't marked as required. @ArekChr mentioned that default props should be defined in destructed props from now. I can return default props to avoid using eslint-disable.

Reply:
@perunt You can bring back defaultProps as it was. Default props will be defined in destructed props while migrating to typescript

Reply:
👍🏻